example/README.md

```diff
@@ -155,6 +155,11 @@ Run `terraform apply`.
 
 This will deploy your data-migration resources.
 
+## Choosing a distribution API
+
+The steps below assume you will be using the [Cumulus Distribution API](./cumulus-tf/cumulus_distribution.tf). If you would prefer
+to use the Thin Egress App (TEA), uncomment the TEA-specific resources in the "cumulus" module in [example/cumulus-tf/main.tf](./cumulus-tf/main.tf).
```

Comment:
If you prefer to use TEA and not the cumulus distribution API, you also have to disable it, correct?

Reply:
You definitely could. My thinking was that, because TEA was the default, if you should choose to deploy TEA instead we'd still deploy the `cumulus_distribution` module but we just wouldn't use it. I guess I was thinking that way because we have integration tests looking for the Cumulus Distribution module and not TEA so your deployment of TEA seemed more likely to be temporary. That only applies to CI deployments from `example` though, you could be deploying using TEA for other testing purposes and then you'd have a useless Cumulus Distribution module taking up space.

🤔 maybe you're right and just a note here to comment it would suffice and cover both bases.

Reply:
You at least need to change the language to clarify that uncommenting TEA will end up deploying both, not one or the other. And you're not really "choosing a distribution API" at that point but just enabling TEA.

Reply:
👍  See what you think of this: https://github.com/nasa/cumulus/pull/2300/commits/dddcc836c7c312f8110060951d759c40b996a92f

Reply:
```suggestion
to use the Thin Egress App (TEA), uncomment the TEA-specific variables in the "cumulus" module in [example/cumulus-tf/main.tf](./cumulus-tf/main.tf).
```

Reply:
Looks good! Just one minor nit

Reply:
Ah yeah, a `resource` is a thing in Terraform. Good call.